Updates basic tutorials for Cadence 1.0 and fixes old migration commands (#112)
onflow/cadence-lang.org: versioned_docs/version-1.0/tutorial/01-first-steps.md

- Resource-oriented programming, a new paradigm that pairs linear types with object capabilities to create a secure and declarative model for digital ownership
  by ensuring that resources (and their associated assets) can only exist in one location at a time, cannot be copied, and cannot be accidentally lost or deleted
- Built-in pre-conditions and post-conditions for functions and [transactions](../language/transactions.md)

> - The utilization of capability-based security and entitlements, which enforces access control by requiring that access to objects

Reviewer: ```suggestion
- The utilization of capability-based security and entitlements, which enforces access control by requiring that access to objects,
```

Author: no comma needed here. We're saying that the access is restricted, not that capability security is restricted

---

The [Flow Playground](https://play.flow.com) includes
an in-browser editor and Flow emulator to experiment with Flow.
Using the Flow Playground, you can write Cadence smart contracts,
deploy them to a local Flow emulated blockchain, and submit transactions.

The Flow Playground should be compatible with any standard web browser,
but we recommend that you use Google Chrome with it,

> because it has been tested and optimized primarily with the Chrome browser so far.

Reviewer: The Flow Playground should work with any standard web browser. However, we recommend using Google Chrome, as it has been primarily tested and optimized for this browser.

The [Flow Crescendo (Cadence 1.0) upgrade](https://flow.com/upgrade/crescendo)

> introduces new powerful features to Cadence. Currently, the playground uses an older version

Reviewer: ```suggestion
introduces new, powerful features to Cadence. Currently, the playground uses an older version
```

The Playground comes pre-loaded with contract and transaction templates
that correspond to each of the tutorials in the docs site.
To load the contracts for a specific tutorial, open the desired tutorial
in the sidebar here and click the link in the callout box at the beginning of the tutorial.

The callout boxes should look like this:

<Callout type="success">
  Open the starter code for this tutorial in the Flow Playground: <br />
  <a
    href="https://play.flow.com/"
    target="_blank"
  >
    https://play.flow.com/
  </a>
</Callout>

When you click on one of these links, the tutorial code will open in a new tab
and the contracts, transactions, and scripts will be loaded 
into the templates in the Playground for you to use.
You will need to navigate between the editor and this tutorial to read instructions
and make changes to your code.

## What is a smart contract?

In regular terms, a contract is an agreement between two parties for some exchange of information or assets.
Normally, the terms of a contract are supervised and enforced by a trusted third party, such as a bank or a lawyer.

A smart contract is a computer program stored in a network like a blockchain

> that verifies and executes the performance of a contract (like a lawyer does)

Reviewer: ```suggestion
that verifies and executes the performance of a contract
```

Author: I disagree with this one. I like the clarification.

without the need for any trusted third party anywhere in the process, because the code itself is trusted.

> Programs that run on blockchains are commonly referred to as smart contracts
> because they mediate important functionality (such as currency)
> without having to rely on a central authority (like a bank).

Reviewer: ```suggestion
Programs that run on blockchains are commonly referred to as smart contracts because they facilitate important functions, such as managing digital currency, without relying on a central authority like a bank.
```
